tools/windows_terminal.bat

```diff
@@ -0,0 +1,6 @@
+:: replace with your distro 
```

Comment:
Could you explain what this does exactly? It sets up a few tabs in Windows Terminal? Seems kinda useful, but maybe it should take the distro as a flag or use the main distro by default.

Reply:
That's correct, it sets up tabs in the working directory that one could use mips_to_c or diff.py in. I double click on the .bat file and it opens the tabs for me. I like that idea, I'll look into setting the default distro automatically.

Reply:
awesome! thanks